jet/sidebar.py

```diff
@@ -0,0 +1,232 @@
+import itertools
+
+import django
+from django.dispatch import Signal
+from django.shortcuts import render
+from django.template import loader, Template, Context
+from django.views.generic.base import TemplateResponseMixin
+from django.utils.functional import cached_property
+from django.utils.safestring import mark_safe
+
+
+class Section(object):
+    """
+    Section base class rendered in Sidebar. Template is cached
+    at a class level.
+    """
+    order = 0
+    """
+    Item order in the sidebar
+    """
+    title = None
+    """
+    Title of the menu
+    """
+    template_name = ''
+    """
+    Template used to render item
+    """
+
+    def __init__(self, **kwargs):
+        self.__dict__.update(kwargs)
+
+    def get_context_data(self, request, page_context):
+        """
+        Return context to use in template. If None is returned, it
+        renders nothing but an empty string.
+        """
+        return {
+            'self': self,
+            'request': request,
+            'ctx': page_context,
+        }
+
+    def render(self, request, page_context):
+        """
+        Render the section
+        """
+        context = self.get_context_data(request, page_context)
+        if context is None:
+            return ''
+        # FIXME: drop support for thoses deprecated versions
+        if django.VERSION <= (1,8):
```

Comment:
Shouldn't this be `1.8` instead of `1,8`?

Reply:
django.VERSION is a tuple, not a floating number